make-up/code/4840-filip-petrovski/src/app/app.routes.ts

```diff
@@ -0,0 +1,32 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { RecipeListComponent } from './recipe-list/recipe-list.component';
+import { RecipeDetailsComponent } from './recipe-details/recipe-details.component';
+import { RecipeEditComponent } from './recipe-edit/recipe-edit.component';
+import { RecipeCreateComponent } from './recipe-create/recipe-create.component';
+import { IngredientsListComponent } from './ingredients-list/ingredients-list.component';
+import { CuisineListComponent } from './cuisine-list/cuisine-list.component';
+import { IngredientsDetailsComponent } from './ingredients-details/ingredients-details.component';
+import { CuisineDetailsComponent } from './cuisine-details/cuisine-details.component';
+import { StatisticsComponent } from './statistics/statistics.component';
+import { AboutComponent } from './about/about.component';
+
+const routes: Routes = [
+  { path: 'recipes', component: RecipeListComponent },
```

Comment:
The routes are defined ok, but they are not exported for usage in the `app.config` +3 points